lib/loader.js

```diff
@@ -64,23 +64,28 @@ module.exports = function(content, map) {
 				"[" + JSON.stringify(importItem.export) + "] + \"";
 		}
 
-		cssAsString = cssAsString.replace(result.importItemRegExpG, importItemMatcher.bind(this)).replace(result.urlItemRegExpG, function(item) {
-			var match = result.urlItemRegExp.exec(item);
-			var idx = +match[1];
-			var urlItem = result.urlItems[idx];
-			var url = urlItem.url;
-			idx = url.indexOf("?#");
-			if(idx < 0) idx = url.indexOf("#");
-			var urlRequest;
-			if(idx > 0) { // idx === 0 is catched by isUrlRequest
-				// in cases like url('webfont.eot?#iefix')
-				urlRequest = url.substr(0, idx);
-				return "\" + require(" + loaderUtils.stringifyRequest(this, urlRequest) + ") + \"" +
-					url.substr(idx);
-			}
-			urlRequest = url;
-			return "\" + require(" + loaderUtils.stringifyRequest(this, urlRequest) + ") + \"";
-		}.bind(this));
+		if(!query.hasOwnProperty('import')) {
```

Comment:
Please don't check for exisitance of query properties. Check for the value. `query.import !== false`. Document it as `css-loader?-url&-import`.

Elsewise this cases would be broken:

* `css-loader?url`
* `css-loader?{url: true}`